components/console-backend-service/Dockerfile

```diff
@@ -1,43 +1,52 @@
-FROM golang:1.11.4-alpine3.8 as builder
+FROM golang:1.13.11-alpine3.11 as builder
```

Comment:
Use golang buildpack instead. It is supposed to hold all the versions we rely upon